uniffi_bindgen/src/bindings/gecko_js/templates/FFIDeclarationsTemplate.h

```diff
@@ -4,11 +4,14 @@ struct {{ context.ffi_rustbuffer_type() }} {
   int32_t mCapacity;
   int32_t mLen;
   uint8_t* mData;
+  uint64_t mPadding;
```

Comment:
nit: I think this should be `int64_t` rather than `uint64_t` (which I don't think should actually matter in practice but hey, who knows...)

Reply:
Good catch; fixed.